pkg/cnb/project_descriptor_test.go

```diff
@@ -243,12 +244,41 @@ value = "valueC"
 name = "keyC"
 value = "valueAnotherC"
 				`), 0644)
+						})
+						it("writes all env var files to the platform dir", func() {
+							assert.Nil(t, cnb.ProcessProjectDescriptor(appDir, descriptorPath, platformDir, logger))
+							checkEnvVar(t, platformDir, "keyA", "valueA")
+							checkEnvVar(t, platformDir, "keyB", "valueB")
+							checkEnvVar(t, platformDir, "keyC", "valueAnotherC")
+						})
 					})
-					it("writes all env var files to the platform dir", func() {
-						assert.Nil(t, cnb.ProcessProjectDescriptor(appDir, descriptorPath, platformDir, logger))
-						checkEnvVar(t, platformDir, "keyA", "valueA")
-						checkEnvVar(t, platformDir, "keyB", "valueB")
-						checkEnvVar(t, platformDir, "keyC", "valueAnotherC")
+
+					when("vars where set with new v0.2 project.toml", func() {
```

Comment:
@jjbustamante can we also add a test case where both are set? The new one should have higher precedence

Reply:
Interesting.

Yes, let me add that case, thanks @samj1912 !